test/helpers.ts

```diff
@@ -65,3 +67,16 @@ export function depTreesEqual(a: any, b: any) {
 
   return true;
 }
+
+export const sortBy = (arr: any[], p: string) =>
```

Comment:
It's totally worth keeping it as a dev dependency just for this.

Reply:
Yeh.. that made me feel funny.

> worth keeping it as a dev dependency

The problem with dev deps that aren't obviously dev deps is that people import them into production code, and all tests pass, but the lib breaks in production